c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -78,6 +78,9 @@ Version 1.6.3
 
 7. Remove the `lang_files` from the `vars/checker` in the file `vars_{{package}}.yaml`.
 
+8. In the PDF report config the show_map attribute will be removed and a map is added in the config hierarchy.
+   See the doc to update your config http://docs.camptocamp.net/c2cgeoportal/master/integrator/pdfreport.html.
```

Comment:
http://docs.camptocamp.net/c2cgeoportal/master/integrator/pdfreport.html is broken as for now.
I would remove the "." at the end of the URL to ease copy-pasting it.